tests/test_processor.py

```diff
@@ -22,10 +23,7 @@ def test_message_processor(default_processor):
 
 
 def test_parsing(default_processor):
-    class Message():
-        text = '/greet{"name":"boy"}'
-
-    message = Message()
+    message = Message('/greet{"name":"boy"}')
```

Comment:
tiny thing, but the examples we're giving have `{"name": "boy"}`, so with a space between the dict's key and value